eclipse-collections-api/src/main/java/org/eclipse/collections/api/factory/bag/MultiReaderBagFactory.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019 Goldman Sachs and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.api.factory.bag;
+
+import java.util.stream.Stream;
+
+import org.eclipse.collections.api.bag.MultiReaderBag;
+
+/**
+ * A factory which creates instances of type {@link MultiReaderBag}.
+ */
+public interface MultiReaderBagFactory
+{
+    /**
+     * @since 6.0
```

Comment:
Add an at since tag for the whole interface and drop the individual at since tags in the methods.

Reply:
Done.  This is included in the latest commit (along with other @since tag changes)